tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -76,10 +75,49 @@ public override bool Execute ()
             File.Copy(Path.Join (MicrosoftNetCoreAppRuntimePackDir, "native", f), Path.Join(AppDir, f), true);
         File.Copy(MainJS!, Path.Join(AppDir, "runtime.js"),  true);
 
-        if (ExtraFiles != null)
+        var filesToMap = new Dictionary<string, List<string>>();
+        if (FilesToIncludeInFileSystem != null)
         {
-            foreach (var item in ExtraFiles)
-                File.Copy(item.ItemSpec, Path.Join(AppDir, Path.GetFileName(item.ItemSpec)),  true);
+            string supportFilesDir = Path.Join(AppDir, "supportFiles");
+            Directory.CreateDirectory(supportFilesDir);
+
+            foreach (var item in FilesToIncludeInFileSystem)
+            {
+                if (item.ItemSpec.EndsWith("deps.json") || item.ItemSpec.EndsWith("runtimeconfig.json"))
```

Comment:
nit: should we filter this out in MSBuild instead? sounds like the more appropriate layer to me (e.g. maybe I want to have a runtimeconfig.json to read for testing purposes in my Wasm app).

Reply:
Ah I had removed this entirely for those reasons but it seems like I didn't commit it. Actually I think it doesn't matter filtering out, it is fine to have them be part of the app bundle + VFS.